devtools/server/actors/replay/connection.js

```diff
@@ -1080,10 +1093,27 @@ function collectUnresolvedSourceMapResources(mapText, mapURL, mapBaseURL) {
 }
 
 async function fetchText(url) {
+  let urlObj;
+  try {
+    urlObj = new URL(url);
+  } catch (err) {
+    urlObj = null;
+  }
+  if (urlObj && !["http:", "https:", "data:"].includes(urlObj.protocol)) {
+    urlObj = null;
+  }
+  if (!urlObj) {
```

Comment:
What conditions lead to no URL? Do we sometime see `file:` protocol urls?

Reply:
Since this is a sourcemap, could be anything the user puts in their build process for instance, like most Webpack maps are `webpack:///foo/bar.js`